### ISSUE TITLE
output scrolls up every 1s  instead of stayin in fixed location.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Patric-R/jvmtop#25]
```
What steps will reproduce the problem?
1.  ./jvmtop.sh
2.
3.

What is the expected output? What do you see instead?
attached jvmtopout.jpg

What version of jvmtop are you using? On what operating system?

jvmtop-0.7.0

Please post the output of the following commands:
java -version
java version "1.6.0_25"
Java(TM) SE Runtime Environment (build 1.6.0_25-b06)
Java HotSpot(TM) 64-Bit Server VM (build 20.0-b11, mixed mode)

echo $JAVA_HOME  (for unix/linux systems)
echo %JAVA_HOME% (for windows)


Please provide any additional information below.

Kindly fix the scrolling mode. it would be nice to have it like linux TOP does. 
 and is it possible to display the output only once ? by which i am planning to 
develop a nagios script plugin which will get output from this and generate 
graphs. 

Awaiting your reply. 
Great work. thanks.
```

Original issue reported on code.google.com by `j...@sodexis.com` on 30 Jul 2013 at 5:38

Attachments:
- [jvmtopout.JPG](https://storage.googleapis.com/google-code-attachments/jvmtop/issue-25/comment-0/jvmtopout.JPG)
